### ISSUE TITLE
Support Argument-Labels

### ISSUE THREAD

**Issue by NiwakaDev** [chinedufn/swift-bridge#153]
Swift's functions have `Argument Labels` and `Parameter Names`. As far as I understand, `swift-bridge` doesn't seem to support `Argument Labels`.

Something like: 
```rust
//Rust
#[swift_bridge::bridge]
mod ffi {
    extern "Rust" {
        #[swift_bridge(argument_labels=(argument_label))]
        fn get(parameter_name: i32);
    }
}

```

```Swift
//Generated Swift 

func get(argument_label parameter_name: Int){

}
```

## Resource
<a href="https://docs.swift.org/swift-book/LanguageGuide/Functions.html">Functions</a>

**Comment by chinedufn** [comment]
Nice, we should totally support this.

As far as the API, what do you think about something like this:

```rust
#[swift_bridge::bridge]
mod ffi {
    extern "Rust" {
        fn greet(
            #[swift_bridge(label = "firstName")] name: &str,
            age: u32
        )
    }
}
```

```swift
func greet(firstName name: RustStr, _ age: u32) {
    // ...
}
```

The argument attribute approach feels a bit cleaner than the function attribute approach.

_Now that I think of it, this is probably a better approach for things like our `get`, `get_with` and `args_into` attribute as well... But we can handle those separately._

**Comment by NiwakaDev** [comment]
>The argument attribute approach makes it a bit easier tell which label applies to which argument.

Sounds good! I didn't know the argument attribute. Can I implement this?

**Comment by chinedufn** [comment]
Go for it!

**Comment by chinedufn** [comment]
Closed via #156
